Fix "error: invalid suffix on literal; C++11 requires a space between literal and identifier [-Wreserved-user-defined-literal]"
--- src/xml.h.orig	2015-04-01 11:12:52
+++ src/xml.h	2024-10-30 05:09:25
@@ -100,7 +100,7 @@ class XML { (public)
     void xmlout(const std::string &tag,const std::string &value){ xmlout(tag,value,"",true); }
     void xmlout(const std::string &tag,const int value){ xmlprintf(tag,"","%d",value); }
     void xmloutl(const std::string &tag,const long value){ xmlprintf(tag,"","%ld",value); }
-    void xmlout(const std::string &tag,const int64_t value){ xmlprintf(tag,"","%"PRId64,value); }
+    void xmlout(const std::string &tag,const int64_t value){ xmlprintf(tag,"","%" PRId64,value); }
     void xmlout(const std::string &tag,const double value){ xmlprintf(tag,"","%f",value); }
     void xmlout(const std::string &tag,const struct timeval &ts){
 	xmlprintf(tag,"","%d.%06d",(int)ts.tv_sec, (int)ts.tv_usec);
--- src/hash.cpp.orig	2024-10-30 05:02:58
+++ src/hash.cpp	2024-10-30 05:11:17
@@ -124,7 +124,7 @@ bool file_data_hasher_t::compute_hash(uint64_t request
     
 	// If an error occured, display a message and see if we need to quit.
 	if ((current_read_bytes<0) || (this->handle && ferror(this->handle))){
-	    ocb->error_filename(this->file_name,"error at offset %"PRIu64": %s",
+	    ocb->error_filename(this->file_name,"error at offset %" PRIu64": %s",
 				request_start, strerror(errno));
 	   
 	    if (file_fatal_error()){
--- src/files.cpp.orig	2015-04-01 11:12:52
+++ src/files.cpp	2024-10-30 05:12:42
@@ -509,7 +509,7 @@ int state::parse_encase_file(const char *fn, FILE *han
         
 	    // Users expect the line numbers to start at one, not zero.
 	    if ((!ocb.opt_silent) || (mode_warn_only)) {
-		ocb.error("%s: No hash found in line %"PRIu32, fn, count + 1);
+		ocb.error("%s: No hash found in line %" PRIu32, fn, count + 1);
 		ocb.error("%s: %s", fn, strerror(errno));
 		return status_t::STATUS_USER_ERROR;
 	    }
@@ -542,7 +542,7 @@ int state::parse_encase_file(const char *fn, FILE *han
     }
 
     if (expected_hashes != count){
-	ocb.error("%s: Expecting %"PRIu32" hashes, found %"PRIu32"\n", 
+	ocb.error("%s: Expecting %" PRIu32 " hashes, found %" PRIu32 "\n", 
 			fn, expected_hashes, count);
     }
     return status_t::status_ok;
--- src/dig.cpp.orig	2015-04-01 11:12:52
+++ src/dig.cpp	2024-10-30 05:13:34
@@ -339,7 +339,7 @@ bool state::is_junction_point(const std::wstring &fn)
 
       default:
 	ocb.error_filename(fn,
-			   "Unknown reparse point 0x%"PRIx32", skipping",
+			   "Unknown reparse point 0x%" PRIx32 ", skipping",
 			   FindFileData.dwReserved0);
 	ocb.error_filename(fn,"Please report this to the developers!");
 
--- src/hashlist.cpp.orig	2015-04-01 11:12:52
+++ src/hashlist.cpp	2024-10-30 05:14:16
@@ -342,7 +342,7 @@ hashlist::load_hash_file(display *ocb,const std::strin
     file_data_t *t = new (std::nothrow) file_data_t();
     if (NULL == t)
     {
-      ocb->fatal_error("%s: Out of memory in line %"PRIu64,
+      ocb->fatal_error("%s: Out of memory in line %" PRIu64,
 		       fn.c_str(), line_number);
     }
 
@@ -390,7 +390,7 @@ hashlist::load_hash_file(display *ocb,const std::strin
       if ( !algorithm_t::valid_hash(hash_column[column_number],word))
       {
 	if (ocb)
-	  ocb->error("%s: Invalid %s hash in line %"PRIu64,
+	  ocb->error("%s: Invalid %s hash in line %" PRIu64,
 		     fn.c_str(),
 		     hashes[hash_column[column_number]].name.c_str(),
 		     line_number);
--- src/display.cpp.orig	2015-04-01 11:12:52
+++ src/display.cpp	2024-10-30 05:15:08
@@ -311,7 +311,7 @@ void display::display_realtime_stats(const file_data_h
 
 	ss << mb_read << "MB of " << fdht->stat_megs() << "MB done, ";
 	char msg[64];
-	snprintf(msg,sizeof(msg),"%02"PRIu64":%02"PRIu64":%02"PRIu64" left", hour, min, seconds);
+	snprintf(msg,sizeof(msg),"%02" PRIu64 ":%02" PRIu64 ":%02" PRIu64 " left", hour, min, seconds);
 	ss << msg;
     }
     ss << "\r";
@@ -424,14 +424,14 @@ void display::display_audit_results()
   
     if (opt_verbose)    {
 	if(opt_verbose >= MORE_VERBOSE){
-	    status("   Input files examined: %"PRIu64, this->match.total);
-	    status("  Known files expecting: %"PRIu64, this->match.expect);
+	    status("   Input files examined: %" PRIu64, this->match.total);
+	    status("  Known files expecting: %" PRIu64, this->match.expect);
 	}
-	status("          Files matched: %"PRIu64, this->match.exact);
-	status("Files partially matched: %"PRIu64, this->match.partial);
-	status("            Files moved: %"PRIu64, this->match.moved);
-	status("        New files found: %"PRIu64, this->match.unknown);
-	status("  Known files not found: %"PRIu64, this->match.unused);
+	status("          Files matched: %" PRIu64, this->match.exact);
+	status("Files partially matched: %" PRIu64, this->match.partial);
+	status("            Files moved: %" PRIu64, this->match.moved);
+	status("        New files found: %" PRIu64, this->match.unknown);
+	status("  Known files not found: %" PRIu64, this->match.unused);
     }
 }
 
